.npmignore

```diff
@@ -0,0 +1,23 @@
+.idea/
+api-doc/
+apps/
+design-notes/
+examples/
+node_modules/
+performance/
+src/
+test/
+test-results/
+tools/
+.gitignore
+.npmignore
+.travis.yml
+config.json
+Gruntfile.js
+HowToBuildWebWW.md
+HowToCreateAndRunUnitTests.md
+karma.conf.js
+layout.tmpl
+package-lock.json
+package.json
```

Comment:
Adding package.json and README.md to .npmignore is unnecessary.
https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package